source/Core/Inc/Settings.h

```diff
@@ -142,7 +143,15 @@ typedef enum {
   // #endif
   TIP_TYPE_MAX, // Max value marker
 } tipType_t;
-uint8_t getUserSelectedTipResistance(); // returns the resistance matching the selected tip type or 0 for auto
+#else
+typedef enum {
+  TIP_TYPE_AUTO = 0, // value for the default case
+  TIP_TYPE_MAX  = 0, // marker for settings when not supported
+} tipType_t;
```

Comment:
Implement duplicated `tipType_t` stub for two reasons:
- `TIP_TYPE_MAX = 0` for [settings struct](https://github.com/Ralim/IronOS/blob/71a1e2a367fcb0f6fd486fd4db2eeaeaa75fdf51/source/Core/Src/Settings.cpp#L112) to avoid compile-time error and set `0/auto` as default right there for non-supported models;
- `TIP_TYPE_AUTO = 0` for backward compatibility with the stub function (see below).